frontend/device/sdl/event_map_sdl2.lua

```diff
@@ -1,39 +1,86 @@
 return {
-    [ 4] = "A", [ 5] = "B", [ 6] = "C", [ 7] = "D", [ 8] = "E", [ 9] = "F",
-    [10] = "G", [11] = "H", [12] = "I", [13] = "J", [14] = "K", [15] = "L",
-    [16] = "M", [17] = "N", [18] = "O", [19] = "P", [20] = "Q", [21] = "R",
-    [22] = "S", [23] = "T", [24] = "U", [25] = "V", [26] = "W", [27] = "X",
-    [28] = "Y", [29] = "Z", [30] = "1", [31] = "2", [32] = "3", [33] = "4",
-    [34] = "5", [35] = "6", [36] = "7", [37] = "8", [38] = "9", [39] = "0",
-
-    [43] = "Tab", -- Tab
-    [42] = "Backspace", -- Backspace
-    [41] = "Back", -- Escape
-    [40] = "Press", -- Enter
-    [225] = "Shift", -- left shift
-    [55] = ".",
-    [56] = "/",
-    [229] = "Sym", -- right shift key
-    [226] = "Alt", -- left alt
-    [44] = " ", -- Spacebar
-    [58] = "Menu", -- F[1]
-    [59] = "Power", -- F[2]
-    [61] = "F4", -- F[4]
-    [63] = "LPgBack", -- F[6]
-    [64] = "LPgFwd", -- F[7]
-    [68] = "VPlus", -- F[11]
-    [69] = "VMinus", -- F[12]
-    [230] = "AA", -- right alt key
-    [74] = "Home", -- Home
-    [82] = "Up", -- arrow up
-    [75] = "RPgBack", -- normal PageUp
-    [80] = "Left", -- arrow left
-    [79] = "Right", -- arrow right
-    [77] = "End", -- End (above arrows)
-    [81] = "Down", -- arrow down
-    [78] = "RPgFwd", -- normal PageDown
-    [76] = "Del", -- Delete
-    [101] = "ContextMenu", -- Context menu key
-    [224] = "Ctrl", -- Left Ctrl
-    [228] = "Ctrl", -- Right Ctrl
+
+    [ 97] = "A", [ 98] = "B", [ 99] = "C", [100] = "D", [101] = "E", [102] = "F",
+    [103] = "G", [104] = "H", [105] = "I", [106] = "J", [107] = "K", [108] = "L",
+    [109] = "M", [110] = "N", [111] = "O", [112] = "P", [113] = "Q", [114] = "R",
+    [115] = "S", [116] = "T", [117] = "U", [118] = "V", [119] = "W", [120] = "X",
+    [121] = "Y", [122] = "Z",
+
+    [48] = "0", [49] = "1", [50] = "2", [51] = "3", [52] = "4",
+    [53] = "5", [54] = "6", [55] = "7", [56] = "8", [57] = "9",
+
+    [33] = "!",
+    [34] = "\"",
+    [35] = "#",
+    [36] = "$",
+    [37] = "#",
+    [38] = "&",
+    [39] = "'",
+    [40] = "(",
+    [41] = ")",
+    [42] = "*",
+    [43] = "+",
+    [44] = ",",
+    [45] = "-",
+    [46] = ".",
+    [47] = "/",
+
+    [58] = ":",
+    [59] = ";",
+    [60] = "<",
+    [61] = "=",
+    [62] = ">",
+    [63] = "?",
+    [64] = "@",
+
+    [91] = "[",
+    [92] = "\\",
+    [93] = "]",
+    [94] = "^",
+    [95] = "_",
+    [96] = "`",
```

Comment:
Should we keep this in? It probably doesn't really matter, but I assume this is a leftover of sorts and we don't really need _all_ the keycode definitions.

Reply:
You might want to access keys in general for something like hotkeys where you don't have SDL text mode open?

Probably you won't see all of them, e.g. ";" and ":" would both be ";" in practice on a normal keyboard, but one has a "Shift/Sym" set and one doesn't.

Reply:
Sure, right now we only use the alphabet for that but I suppose you mean in a hypothetical user configurable sense? Let's just keep it in then.